R/transition_risk_score.R

```diff
@@ -0,0 +1,126 @@
+#' Transition Risk Score
+#'
+#' Calulate Transition Risk Score at product level and company level
+#'
+#' @param emissions_profile_at_product_level Dataframe. Emissions profile product level output
+#' @param sector_profile_at_product_level Dataframe. Sector profile product level output
+#'
+#' @family top-level functions
+#'
+#' @export
+#'
+#' @examples
+#' library(dplyr)
+#' library(readr, warn.conflicts = FALSE)
+#'
+#' emissions_profile_at_product_level <- example_emissions_profile_at_product_level()
+#' sector_profile_at_product_level <- example_sector_profile_at_product_level()
```

Comment:
Above I see these functions are internal. Internal functions are best hidden from users so we can change them without deprecation. If we need to expose them to users, then I wonder why we want them to be internal?

Reply:
Great, your intention is clearer now. 

If `i()` is internal but `p()` is public but shows `i()` in `@examples` then the user can't see the helpfile of `i()` but can see its usage in the helpfile of `p()` (and therefore on the website).

I lean towards using `i()` only in anything internal to remove duplication, but tolerate a bit of duplication in the public helpfile of `p()`.

Reply:
I have changed the example so that now the user can see the public toy datasets. Please review again! Thanks :)